docker/Dockerfile

```diff
@@ -10,5 +10,7 @@ ADD . /usr/src/schedule-utilities
 
 RUN npm install
 
+RUN npm run build
```

Comment:
I don't know how CI builds ran before, but the npm `prepare` script fails to run in all of the builds I ran with `prepare: cannot run in wd schedule-utilities`. So we need to call `npm run build` in the dockerfile explicitly after `npm install` rather than relying on npm postinstall hook.